<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appetizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListView,
QTableView {
    show-decoration-selected: 1; /* make the selection span the entire width of the view */
    selection-background-color:  rgba(81, 179, 35, 50)
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1366</width>
      <height>767</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QComboBox" name="lista_categorias">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>452</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mesero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Host</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cocinero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Manager</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cajero</string>
       </property>
      </item>
     </widget>
     <widget class="KeypadWidget" name="keypad" native="true">
      <property name="geometry">
       <rect>
        <x>774</x>
        <y>108</y>
        <width>480</width>
        <height>397</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QListView" name="userListView">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>162</y>
        <width>452</width>
        <height>361</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="welcomePage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>504</x>
        <y>234</y>
        <width>253</width>
        <height>109</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bienvenido!!</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeypadWidget</class>
   <extends>QWidget</extends>
   <header>keypadwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
